package.json

```diff
@@ -7,6 +7,9 @@
     "dev": "nodemon --exec dotenv ts-node src/index.ts",
     "test": "NODE_ENV=test jest",
     "coverage": "yarn codecov",
+    "build": "tsc -p tsconfig.json",
+    "start": "node -r dotenv/config dist/index.js",
+    "postinstall": "npm run build",
```

Comment:
heroku is now building projects automatically if I'm not mistaken https://devcenter.heroku.com/changelog-items/1557

Reply:
Great, I'm removing that commit to clean up the history. I wasn't aware of that change in Heroku's policy.